tests/NHSD.BuyingCatalogue.Testing.Data/SqlRunner.cs

```diff
@@ -40,10 +40,8 @@ internal static async Task<IEnumerable<T>> FetchAllAsync<T>(string selectSql, ob
 
         internal static async Task<T> QueryFirstAsync<T>(string sql, object parameters = null)
         {
-            using (IDbConnection databaseConnection = new SqlConnection(ConnectionStrings.GPitFuturesSetup))
-            {
-                return (await databaseConnection.QueryFirstAsync<T>(sql, parameters));
-            }
+            using IDbConnection databaseConnection = new SqlConnection(ConnectionStrings.GPitFuturesSetup);
+            return (await databaseConnection.QueryFirstAsync<T>(sql, parameters));
```

Comment:
Sorry, missed this last time: the brackets aren't required around the `await`.

Reply:
👍